src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The bulldozer started working");

```

Comment:
The message should indicate that it is a Bulldozer that is working. Consider changing the message to 'Bulldozer started working' to make it more informative as per the checklist.